src/main/java/seedu/duke/ui/MessageBubble.java

```diff
@@ -7,8 +7,8 @@
     static final String EDGE_VERTICAL_SYMBOL = ":";
     static final String EDGE_TAIL = "...";
     static int MAX_LINES = 1000;
-    static int TERMINAL_WIDTH = 80;
-    static int MAX_BUBBLE_WIDTH = 70;
+    static int TERMINAL_WIDTH = 105;
+    static int MAX_BUBBLE_WIDTH = 105;
```

Comment:
Try to set the MAX_BUBBLE_WIDTH smaller than the terminal width so that the lefthand side of the bubble will not touch the left side of the terminal.